shell/platform/windows/flutter_platform_node_delegate_windows.cc

```diff
@@ -62,9 +62,10 @@ gfx::NativeViewAccessible FlutterPlatformNodeDelegateWindows::HitTestSync(
   auto bridge = bridge_.lock();
   FML_DCHECK(bridge);
   for (const ui::AXNode* child : GetAXNode()->children()) {
-    std::shared_ptr<FlutterPlatformNodeDelegateWindows> win_delegate =
-        std::static_pointer_cast<FlutterPlatformNodeDelegateWindows>(
-            bridge->GetFlutterPlatformNodeDelegateFromID(child->id()).lock());
+    std::shared_ptr<FlutterPlatformNodeDelegate> delegate =
```

Comment:
Is the existence of the `delegate` variable what makes the difference? And is it only used to cast to `win_delegate`? It looks like something the compiler might optimize away before the scope actually exits anyway, so are we sure this actually makes a difference?